_includes/footer.md

```diff
@@ -4,10 +4,7 @@
 [![Apache Software Foundation][ASF_IMG]{: #asf-logo height="100"}][ASF]
 
 Copyright &copy; 2011-{{ site.time | date: '%Y' }} The Apache Software
-Foundation, Licensed under the [Apache License, Version 2.0][AL2]. Apache
```

Comment:
Is this even required? It's messy to have it and is cleaner without, and I'm skeptical that it actually does anything to further protect the Apache brand (we already have "Apache" prefixes everwhere). So, if it's not required (and I can't find anything that says it is), I'd prefer it trimmed down.